### PR TITLE
1.7.10

### PR DESCRIPTION
**Pull request by PseudoDistant**
The heckin 1.7.10 API

### CODE REVIEW

**Comment by PseudoDistant** [comment]
Fack

**Comment by PseudoDistant** [comment]
Fixed

File: HEADER [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2021 Legacy Rewoven
```
Comment (Grayray75) [inline HEADER:1]: remove that header?

Comment (PseudoDistant) [reply HEADER:1]: Ack, so much changing.
The Voices

Comment (PseudoDistant) [reply HEADER:1]: > Content length exceeded
https://tenor.com/view/the-voices-meme-cat-gif-23917781

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+Legacy Fabric API is the library for essential hooks and interoperability mechanisms for Legacy Fabric mods. Examples include:
 
-Exposing functionality that is useful but difficult to access for many mods such as particles, biomes and dimensions
-Adding events, hooks and APIs to improve interopability between mods.
+- Exposing functionality that is useful but difficult to access for many mods such as biomes and enchantments
+- Adding events, hooks, APIs and commonly used hacks to improve interoperability between mods such as resource loading, networking and gamerules
+- Adding APIs that help circumvent difficult-to-use minecraft APIs such as commands
 
-For support, consider joining the [Legacy Fabric discord server](https://legacyfabric.net/discord) or opening an issue.
+For support, consider joining the [Legacy Fabric discord server](https://discord.gg/emgF7bp), opening an issue, or creating a discussion.
```
Comment (Grayray75) [inline README.md:11]: why change the discord link?

Comment (PseudoDistant) [reply README.md:11]: I refactored Rewoven API, and the link in that was Rewoven's Discord.
The change there is unintentional, but also completely unimpactful as it's the same result anyway.

File: HEADER (hunk 1 above)
Comment (PseudoDistant) [reply HEADER]: Finished

File: README.md (hunk 2 above)
Comment (PseudoDistant) [reply README.md]: Also finished.

File: testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Keybindings
```
Comment (arthurbambou) [inline testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: you forgot to derewoven this file

**Review by arthurbambou — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 4]
```diff
@@ -4,7 +4,7 @@ plugins {
 	id 'idea'
 	id 'maven-publish'
 	id 'fabric-loom' version "0.7-SNAPSHOT"
```
[the remark this answers is not in the document]
Comment (PseudoDistant) [reply build.gradle:6]: I actually tried for a bit, then decided to do it later lole .

File: modules/networking/src/main/resources/fabric.mod.json [hunk 5]
```diff
@@ -5,15 +5,16 @@
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-fabric/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyfabric.net",
+    "homepage": "https://legacy-fabric.github.io/",
```
[the remark this answers is not in the document]
Comment (PseudoDistant) [reply modules/networking/src/main/resources/fabric.mod.json:10]: Oof, accident lol

File: testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang (hunk 3 above)
Comment (PseudoDistant) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: No that's intentional.

Comment (arthurbambou) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: ok

**Review by arthurbambou — APPROVED** [review]

File: testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang (hunk 3 above)
Comment (PseudoDistant) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: Wait there's supposed to be a second one in the right folder.

Comment (PseudoDistant) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: That was meant to be left in there, but there should also be a proper one.
Facking IDEA facking deleting my shit. ;-;

Comment (arthurbambou) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: concern

Comment (PseudoDistant) [reply testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang:1]: Should be good now.

**Review by arthurbambou — APPROVED** [review]

**Pull request merged**